frontend/packages/kubevirt-plugin/integration-tests/tests/utils/utils.ts

```diff
@@ -51,7 +51,7 @@ export async function createProject(name: string) {
 }
 
 export async function createExampleVMViaYAML() {
```

Comment:
I think it would be better to refactor createExampleVMViaYAML to return VirtualMachine object instance.
You could then re-use all the VirtualMachine navigation and other useful method (which I refer to comments later)

`return new VirtualMachine({ name: DEFAULT_YAML_VM_NAME, namespace: testName });`